<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LayerProperties</class>
 <widget class="QWidget" name="LayerProperties">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>298</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Layer Properties</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <property name="horizontalSpacing">
      <number>3</number>
     </property>
     <property name="verticalSpacing">
      <number>5</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="titleLabel">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="title"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="opacityLabel">
       <property name="text">
        <string>Opacity:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QSlider" name="opacitySlider">
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="opacitySpinner">
         <property name="suffix">
          <string>%</string>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="blendModeLabel">
       <property name="text">
        <string>Blend Mode:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="blendMode"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="visible">
     <property name="text">
      <string>Visible</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="fixed">
     <property name="text">
      <string>Fixed</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="clippingGroup">
     <property name="text">
      <string>Clipping Group</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="defaultLayer">
     <property name="text">
      <string>Default</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>opacitySlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>opacitySpinner</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>154</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>259</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>opacitySpinner</sender>
   <signal>valueChanged(int)</signal>
   <receiver>opacitySlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>259</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>154</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
